tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1349,12 +1349,15 @@ def add_nexthop_routes(standby_tor, route_dst, nexthops=None):
     logging.info("Route added to {}: {}".format(standby_tor.hostname, route_cmd))
 
 
-def remove_static_routes(standby_tor, active_tor_loopback_ip):
+def remove_static_routes(standby_tor, route_dst):
     """
     Remove static routes for active tor
```

Comment:
minor: The comment says that routes are removed on active tor, however code removes it from standby tor